fleet-management-persistence/src/main/java/cz/fi/muni/pa165/entity/VehicleCategory.java

```diff
@@ -15,7 +15,6 @@
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
-    @NotNull
```

Comment:
do databazy ale aj tak nemozes vlozit null name, vyhodi to DataAccessException je nato aj test :)